packages/fxa-shared/db/models/auth/linked-account.ts

```diff
@@ -11,6 +11,11 @@ export const PROVIDER = {
   apple: 2,
 } as const;
 export type Provider = keyof typeof PROVIDER;
+export const PROVIDER_NAME = {
+  __fxa__unmapped: 'Unknown',
```

Comment:
Doesn't look like we actually use/set the default anywhere for email bounces. I think it is probably ok because it would help avoid anything getting into the db that is unexpected.